Corrected nats documentation

girishg4t opened the pull request:
Made changes to the nats documentation so that the keda connector should work by following the steps

RealHarshThakur commented:
@girishg4t  
Some nitpicks(Mostly related to assuming people will not clone the repo): 
* Can you please provide links to the YAML files ?
* Can you provide the Dockerfile , Go files as code snippets in the docs, so people can copy , paste ?
* I think `deployment.yaml` is missing in the repo too, can you please check?

girishg4t commented:
> Some minor changes:
> 
>     * Can you add the Dockerfile in a code snippet too?
> 
>     * Can you provide the link to deployment.yaml in kubectl command?

1) added dockerfile
2) Also added link to deplyment.yaml and code snippit

RealHarshThakur commented on docs/content/en/docs/triggers/message-queue-trigger-kind-keda/nats-streaming.md:
````diff
@@ -31,26 +23,75 @@ NATS streaming keda connector uses NATS monitoring to scale the deployment, to e
-```bash
-$ curl http://localhost:8222/streaming/channelsz?channel=request
+```sh
+$ kubectl apply -f nats-dep.yaml
````
```suggestion
$ kubectl apply -f https://github.com/fission/keda-connectors/blob/master/nats-streaming-http-connector/test/nats-streaming-server/nats-dep.yaml
```

RealHarshThakur commented:
Thanks @girishg4t

sanketsudake commented:
Closing this PR as changes got merged via https://github.com/fission/docs.fission.io/pull/294
